frontend/src/pages/application/Application.js

```diff
@@ -108,19 +115,28 @@ export default () => {
     const closeModalAddShipment = (isOpen, appDto) => {
         setModalAddShipmentOpen(isOpen);
         if (appDto) {
-            getApplications(`/customers/${customerId}/application?page=${page.currentPage}&size=${page.countPerPage}&isAll=${isAll}`);
+            getApplications(`/customers/${customerId}/application?page=${page.currentPage}&size=${page.countPerPage}&isAll=${isAll}&status=${filter}`);
         }
     };
 
     const closeModalEdit = (e) => {
         setOpenEditModal(e);
+        getApplications(`/customers/${customerId}/application?page=${page.currentPage}&size=${page.countPerPage}&isAll=${isAll}&status=${filter}`);
+
     };
 
     const closeModalEditShipment = (e) => {
         setOpenEditShipmentModal(e);
+        getApplications(`/customers/${customerId}/application?page=${page.currentPage}&size=${page.countPerPage}&isAll=${isAll}&status=${filter}`);
+    };
+
+    const closeModalAccept = (isOpen) => {
+        setModalAcceptOpen(isOpen);
+        getApplications(`/customers/${customerId}/application?page=${page.currentPage}&size=${page.countPerPage}&isAll=${isAll}&status=${filter}`);
     };
 
-    const tableRows = applications.map(app => (
+
+    const tableRows = applications && applications.map(app => (
```

Comment:
How will tableRows look like if applications object is empty?

Reply:
![image](https://user-images.githubusercontent.com/38259449/100861933-8922b980-34a3-11eb-8d37-12fec28246f5.png)
I'll remove first (applications) check